### ISSUE TITLE
NS Vue: Can't disable zoom controls on android

### ISSUE THREAD

**Issue by gitllermopalafox** [Notalib/nativescript-webview-ext#22]
Hi, thanks for your great work!

I'm using NS Vue and I can not disable Android's default zoom controls.

I tried these options but none works:

```
            <WebViewExt
                    row="0"
                    src="~/assets/www/index.html"
                    builtInZoomControls="false"
                    displayZoomControls="false"
                    supportZoom="false"

            ></WebViewExt>
```

![image](https://user-images.githubusercontent.com/4684919/53626319-00d6fc80-3bcb-11e9-91ab-6934430c8175.png)

### Which platform(s) does your issue occur on?
Platform: Android
Tested on physical and emulator phone
Physical phone: Samsung Galaxy A8+ (SM-A730F) Android 8
Emulator: Pixel XL API 28 Android 9

### Please, provide the following version numbers that your issue occurs with:
NS version: 5.2.0
NS Vue version: 2.0.2
NodeJs version: 11.6.0

#### Example project:
https://github.com/gitllermopalafox/WebviewExtNSVueTest

**Comment by m-abs** [comment]
Hi @gitllermopalafox,

Thank you very much for the bug rapport.
I'm able to reproduce the issue and it'll be fixed in the next release.

**Comment by m-abs** [comment]
Hi again,

The problem should now be fixed in our master branch.

But you'll have to wait til next week before I can run tests before releasing a new version.

You might be able to work around the problem until then.
The problem is caused by a missing value converter, so the "false" value is treated as a string.

If you can use the boolean value it should work.

I don't know how you do that from template in Vue. :) 
In angular it is very easy, just write it like this: 
```html
          <WebViewExt
                    row="0"
                    src="~/assets/www/index.html"
                    [builtInZoomControls]="false"
                    [displayZoomControls]="false"
                    [supportZoom]="false"
            ></WebViewExt>
```

**Comment by gitllermopalafox** [comment]
Hi, thanks for your answers.

I think the closest to do same as angular is using "v-bind" to pass a js expression, just add ":" behind the property.

https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean

```
<WebViewExt
                    row="0"
                    src="~/assets/www/index.html"
                    :builtInZoomControls="false"
                    :displayZoomControls="false"
                    :supportZoom="false"
            ></WebViewExt>
```

* I tried with "false" and It didn't work.
* I tried with "0" and it works. 😀

**Comment by gitllermopalafox** [comment]
Hi, thanks for your answers.

I think the closest to do same as angular is using "v-bind" to pass a js expression:

https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean

```
<WebViewExt
                    row="0"
                    src="~/assets/www/index.html"
                    :builtInZoomControls="false"
                    :displayZoomControls="false"
                    :supportZoom="false"
            ></WebViewExt>
```

* I tried with "false", It didn't work.
* I tried with "0", It didn't work.

I'll try different approaches.

**Comment by m-abs** [comment]
Hi @gitllermopalafox 

I've just released version 5.1.0, that should solve the problem.

**Comment by gitllermopalafox** [comment]
Hi @m-abs thanks a lot. I can confirm the fix is working correctly.
